cvat-core/src/annotations-saver.js

```diff
@@ -102,12 +102,18 @@
                 },
             };
 
+            const keys = ['id', 'label_id', 'group', 'frame',
```

Comment:
Let's create a function to covert an object into "hash". Thus you don't need to repeat keys in several files.

Reply:
@nmanovic 
The first file from legacy UI,
The second from cvat-core.

They don't related anyhow.